Header.c

```
@@ -59,24 +63,49 @@ void Header_setLayout(Header* this, HeaderLayout hLayout) {
+      Vector_remove(lastColumn, Vector_size(lastColumn) - 1);
+   }
+   lastColumn->owner = true;
+}
+
+void Header_collapseLayout(Header* this) {
+   size_t currentColumns = HeaderLayout_getColumns(this->headerLayout);
+   size_t maxColumns = this->maxColumns;
+
+   assert(this->metersCopied || currentColumns == this->maxColumns);
+
+   for (size_t i = currentColumns; i < maxColumns; i++) {
```

Cppcheck (2.20.0)

[style] The function 'Platform_longOptionsUsage' is never used.

(unusedFunction)